packages/client/src/context.ts

```diff
@@ -71,7 +71,7 @@ export class Context {
   /**
    * Load a document by ID, using the cache if possible.
    */
-  async loadDoc<Content = Record<string, any>>(
+  async loadDoc<Content extends Record<string, any> = Record<string, any>>(
```

Comment:
Are we using the possibility to extend `Record<sting, any>` anywhere?

Reply:
It's needed with TS 4.8 being more strict about types, as `TileDocument` defines its contents as `Record<string, any>`, we need to specify the `Content` type here implements this type.

Reply:
Thanks :)